### ISSUE TITLE
@distube/ytdl-core is out of date on release 2.9.1

### ISSUE THREAD

**Issue by DeadlyVirez** [museofficial/muse#1060]
**What's happening?**
@distube/ytdl-core is out of date, but the playback seems to work

**Logs**

```
muse-1  | WARNING: @distube/ytdl-core is out of date! Update with "npm install @distube/ytdl-core@latest".
```

**Screenshots**
![grafik](https://github.com/user-attachments/assets/16da1f50-2641-4d26-b83e-11b4d8c65979)

**Runtime**
I'm running Muse:
- [ ] Directly from the cloned repository
- [x] Inside a Docker container
- [ ] Something else (please elaborate)

**Versions**
- Muse: 2.9.1
- Docker (if applicable): 27.1.1
- OS: Ubuntu 24.04 LTS

**Comment by Cosmo121** [comment]
Same error here.

**Comment by REALJasonAU** [comment]
Same here

**Comment by xKosar** [comment]
same issue

**Comment by asusplayer** [comment]
> When updating the package inside the container, playback breaks

Seems to be working for me, had no playback before updating, after running npm update inside of the container playback works again

**Comment by gacrestani** [comment]
Same error here.

**Comment by lilkingjr1** [comment]
Same issue for me, but running `npm install @distube/ytdl-core@latest` fixed it for me. Not sure why `yarn install` from a fresh install would not download the right package?

**Comment by Cosmo121** [comment]
@DeadlyVirez can confirm, but this appears to be fixed in the latest release.

**Comment by lilkingjr1** [comment]
Latest release resolves this for me too. Thank you ❤️

**Comment by Hazzajenko** [comment]
Closing as this was fixed in [Release v2.9.3](https://github.com/museofficial/muse/releases/tag/v2.9.3).
